tests/Base.php

```diff
@@ -462,6 +462,85 @@ public function testDeprecatedMethods(): void
         self::assertEquals('{"hello":"world"}', $response['body']);
     }
 
+    public function testResponseChunkedSimple(): void
+    {
+        $body = '';
+        $chunks = 0;
+        $response = Client::execute(body: 'Hello', headers: ['x-action' => 'responseChunkedSimple'], callback: function($chunk) use(&$body, &$chunks) {
+            $body .= $chunk;
+            $chunks += 1;
+        });
+
+        self::assertEquals(200, $response['code']);
+        self::assertEquals('OK1OK2', $body);
+        self::assertGreaterThanOrEqual(2, $chunks);
+        self::assertEmpty(Client::getLogs($response['headers']['x-open-runtimes-log-id']));
+        self::assertEmpty(Client::getErrors($response['headers']['x-open-runtimes-log-id']));
+    }
+
+    public function testResponseChunkedComplex(): void
+    {
+        $body = '';
+        $chunks = 0;
+        $response = Client::execute(body: 'Hello', headers: ['x-action' => 'responseChunkedComplex'], callback: function($chunk) use(&$body, &$chunks) {
+            $body .= $chunk;
+            $chunks += 1;
```

Comment:
Not sure if this is enough. Would be more confident knowing it took ~1s between each chunk, to ensure sleeping in-between works, and you still get all data, and you get it gradually (not all chunks once at the end)